src/app/components/DynamicProgress.js

```diff
@@ -1,38 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
-import { Progress, Icon } from 'react-components';
+import { Icon } from 'react-components';
 import contactLoader from 'design-system/assets/img/pm-images/contact-export-loader.gif';
 
-const DyanmicProgress = ({ id, alt, displayDuring, displayEnd, value, max, ...rest }) => {
+const DynamicProgress = ({ id, alt, displayDuring, displayEnd, value, max, endPostponed, ...rest }) => {
     return (
         <div className="aligncenter">
             {value < max ? (
                 <img src={contactLoader} alt={`${alt}-loader`} />
             ) : (
                 <Icon name="on" fill="success" size={100} alt={`${alt}-finished`} />
             )}
-            <Progress className="progress-contact w100 mt1" aria-describedby={id} value={value} max={max} {...rest} />
+            <progress className="progress-contact w100 mt1" aria-describedby={id} value={value} max={max} {...rest} />
             <p aria-atomic="true" aria-live="polite" id="id">
-                {value < max ? `${displayDuring}: ${value}%` : displayEnd}
+                {value < max || endPostponed ? `${displayDuring}: ${value}%` : displayEnd}
             </p>
         </div>
     );
 };
 
-DyanmicProgress.propTypes = {
+DynamicProgress.propTypes = {
     id: PropTypes.string.isRequired,
     alt: PropTypes.string.isRequired,
     displayDuring: PropTypes.string,
     displayEnd: PropTypes.string,
     value: PropTypes.number.isRequired,
-    max: PropTypes.number.isRequired
+    max: PropTypes.number.isRequired,
+    endPostponed: PropTypes.bool.isRequired
 };
 
-DyanmicProgress.defaultProps = {
+DynamicProgress.defaultProps = {
     displayDuring: c('Progress bar description').t`Progress`,
```

Comment:
Have considered using default arguments values instead of defaultProps?

Reply:
in the React projects we're sticking to specifying default props via `defaultProps`. There are some limit cases in which `defaultProps` would be preferred, although in general they could be interchangeable.